moreores.lua

```diff
@@ -30,11 +15,8 @@ end
 
 if bitchange.use_default_tin then
 	minetest.register_craft({
+		type   = "cooking",
 		output = "bitchange:mineninth 18",
```

Comment:
Shouldn't the output also halve?

Reply:
Right now it's consistent with the amount of coins you get from gold